packages/angular/src/selector/selector.component.html

```diff
@@ -12,16 +12,31 @@
  <!-- !model.completed || model.saving -->
  <mat-progress-bar mode="indeterminate" *ngIf="!styled"></mat-progress-bar>
 
+<!-- This shows up if the list is empty -->
+ <ng-template #emptyList>
```

Comment:
This `ng-template` has an id of `emptyList` and is rendered wih the `*ngIf="!empty; else emptyList"` on line `29`.